<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Éditeur de Graphes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_main">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>

    <!-- Zone du graphe -->
    <item>
     <layout class="QVBoxLayout" name="grapheLayout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>

    <!-- Panneau de contrôle -->
    <item>
     <widget class="QGroupBox" name="controlGroupBox">
      <property name="title">
       <string>Contrôles</string>
      </property>
      <property name="minimumHeight">
       <number>120</number>
      </property>
      <layout class="QVBoxLayout" name="controlLayout">
       <property name="spacing">
        <number>10</number>
       </property>

       <!-- Ligne 1: Génération du graphe -->
       <item>
        <layout class="QHBoxLayout" name="generationLayout">
         <item>
          <widget class="QLabel" name="generationLabel">
           <property name="text">
            <string>Graphe :</string>
           </property>
           <property name="minimumWidth">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="createButton">
           <property name="text">
            <string>Générer</string>
           </property>
           <property name="minimumWidth">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="text">
            <string>Effacer</string>
           </property>
           <property name="minimumWidth">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>

       <!-- Ligne 2: Gestion de l'arête -->
       <item>
        <layout class="QHBoxLayout" name="edgeLayout">
         <item>
          <widget class="QLabel" name="edgeTitleLabel">
           <property name="text">
            <string>Arête :</string>
           </property>
           <property name="minimumWidth">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="edgeLabel">
           <property name="text">
            <string>Aucune sélection</string>
           </property>
           <property name="minimumWidth">
            <number>150</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="weightLabel">
           <property name="text">
            <string>Poids :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="weightSpinBox">
           <property name="minimumWidth">
            <number>80</number>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>

      </layout>
     </widget>
    </item>

   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>